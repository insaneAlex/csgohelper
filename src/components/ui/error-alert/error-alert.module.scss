@import 'styles/index';

.alert {
  margin: 1rem auto;
  padding: 1rem 2rem;
  width: 90%;
  max-width: 40rem;
  background-color: $dark-red;
  font-weight: bold;
  font-size: 1.2rem;
  text-align: center;
  border-radius: 6px;

  @include body-short;
}
