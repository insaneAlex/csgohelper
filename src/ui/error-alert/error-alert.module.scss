@import 'styles/variables.scss';

.alert {
  margin: 1rem auto;
  padding: 1rem 2rem;
  width: 90%;
  max-width: 40rem;
  background-color: #b81414;
  color: $main-font-color;
  font-weight: bold;
  font-size: 1.5rem;
  text-align: center;
  border-radius: 6px;
}
