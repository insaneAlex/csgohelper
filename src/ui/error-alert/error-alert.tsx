import {FC, ReactNode} from 'react';
import styles from './error-alert.module.scss';

export const ErrorAlert: FC<{children: ReactNode}> = ({children}) => <div className={styles.alert}>{children}</div>;
