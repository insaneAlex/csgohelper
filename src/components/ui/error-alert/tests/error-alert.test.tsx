import {render} from '@testing-library/react';
import {ErrorAlert} from '../error-alert';

describe('ErrorAlert', () => {
  it('should render correctly', () => {
    const {container} = render(<ErrorAlert>Some error</ErrorAlert>);
    expect(container).toMatchSnapshot();
  });
});
