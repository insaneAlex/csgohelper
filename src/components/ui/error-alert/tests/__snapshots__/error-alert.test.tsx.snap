// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorAlert should render correctly 1`] = `
<div>
  <div
    class="alert"
  >
    Some error
  </div>
</div>
`;
